python/lsst/consdb/pqserver.py

```diff
@@ -181,6 +182,26 @@ def get_timestamp_columns(self, table: str) -> set[str]:
         columns = self.timestamp_columns[table]
         return columns
 
+    def get_schema_version(self, instrument: str) -> Version:
+        if "day_obs" in self.schemas[instrument].tables[f"cdb_{instrument}.ccdexposure"].columns:
+            return Version("3.2.0")
```

Comment:
What happens if we are on 3.3.0, would we have to change this every time we upgrade?

Reply:
My hope is that Jeremy will modify felis so that the version number is stored in the database. We can then use a database query to get the version, with maybe something like the above as a fallback when the version number is not present. As we leave behind older schemas with no versioning in the database, this would become unnecessary.